src/cli/connectSdkGenerator.js

```diff
@@ -7,6 +7,11 @@ async function generateSDK() {
          path: 'https://cloud.handcash.io/sdk-docs.json',
       },
       output: path.join(process.cwd(), 'src/client'),
+      operations: {
+         exclude: [
```

Comment:
Would it make sense to follow the opposite approach? (specific list of things to include)

Reply:
Whitelisting sounds more "defensive" from the sense that it doesn't care if new routes are added.

I suppose both approaches are effective at this point.